app/models/friendship.rb

```diff
@@ -1,4 +1,41 @@
 class Friendship < ActiveRecord::Base
+  after_initialize :set_defaults, unless: :persisted?
+  after_create :create_inverse, if: :inverse_record_nil?
+  after_update :update_inverse
+  after_destroy :destroy_inverse
+
   belongs_to :user
   belongs_to :friend, class_name: 'User'
+
+  def set_defaults
+    self.confirmed = false if confirmed.nil?
+  end
+
+  def create_inverse
+    # Friendship.create(user: user, friend: friend, confirmed: nil)
```

Comment:
Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.